Src/Test.UnitTests.Sarif.PatternMatcher.Function/HttpAnalyzeFunctionTests.cs

```diff
@@ -106,5 +92,54 @@ public async Task Function_HttpAnalyze_WithEmptyFileName_Should_Return_OK()
             resultObject.Should().NotBeNull();
             resultObject.StatusCode.Should().Be((int)HttpStatusCode.OK);
         }
+
+        [Fact]
+        public async Task Function_HttpAnalyze_WithFileContent_Should_Return_DifferentResponses()
+        {
+            const string patTextFile = "SEC101_005.SlackTokens.py";
+            string content = TestHelper.GetTestResourceContent(patTextFile);
+            string[] lines = content.Split(Environment.NewLine);
+
+            IActionResult result = await HttpAnalyzeFunction.Analyze(
+                request: TestHelper.MockAnalyzeFunctionRequest(patTextFile, lines[0]),
+                log: logger,
+                context: TestHelper.ContextSetup());
+            ValidateResult(lines[0], result, runCount: 1, resultCount: 1, FailureLevel.Warning);
+
+            result = await HttpAnalyzeFunction.Analyze(
+               request: TestHelper.MockAnalyzeFunctionRequest(patTextFile, lines[1]),
+               log: logger,
+               context: TestHelper.ContextSetup());
+            ValidateResult(lines[1], result, runCount: 1, resultCount: 0, FailureLevel.None);
```

Comment:
Specifying "FailureLevel.None" is redundant here, isn't it?

Reply:
yes! let me remove! :)

---
In reply to: [566388008](https://github.com/microsoft/sarif-pattern-matcher/pull/138#discussion_r566388008) [](ancestors = 566388008)